Dockerfile.tests

```diff
@@ -1,7 +1,8 @@
 FROM ubuntu:18.04 AS build
-RUN apt update && apt install -y automake autotools-dev make gcc libglib2.0-dev libcap-dev libseccomp-dev
+RUN apt update && apt install -y automake autotools-dev make gcc libglib2.0-dev libcap-dev libseccomp-dev git meson
```

Comment:
I see `make indent` is still failing.
Maybe `meson` dpkg comes with something that breaks the default clang-format configuration?
Does `apt-get install --no-install-recommends` work?

Reply:
Or `apt-get remove meson` before `CMD ["make", "ci"]`